test/integration/scripts/create_pkcs_store.sh

```diff
@@ -56,8 +56,9 @@ set -e
 tpm2_ptool.py init --pobj-pin=mypobjpin --path=$TPM2_PKCS11_STORE
 
 # Test the existing primary object init functionality
-tpm2_createprimary -p foopass -o primary.ctx -g sha256 -G rsa
-handle=`tpm2_evictcontrol -a o -c primary.ctx | cut -d\: -f2-2 | sed 's/^ *//g'`
+primary_context=`mktemp`
+tpm2_createprimary -p foopass -o "$primary_context" -g sha256 -G rsa
+handle=`tpm2_evictcontrol -a o -c "$primary_context" | cut -d\: -f2-2 | sed 's/^ *//g'`
```

Comment:
we should probably do this as $TPM2_PKCS11_STORE/primary.ctx
so an rm -rf of TPM2_PKCS11_STORE deletes everything.

Reply:
Thanks, that makes more sense indeed, implemented!